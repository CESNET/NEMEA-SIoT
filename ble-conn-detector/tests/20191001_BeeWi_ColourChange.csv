time TIMESTAMP,macaddr DEV_ADDR,uint8 ATYPE,int8 RSSI
2033-01-11T03:45:36.000,07:00:00:cc:86:af,32,118
2019-09-25T10:25:38.732,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:38.840,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:25:38.942,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:39.043,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:25:39.143,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:39.244,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:39.352,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:39.457,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:39.558,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:39.666,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:39.769,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:39.868,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:39.969,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:40.070,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:40.175,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:40.280,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:40.386,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:25:40.487,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:40.594,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:25:40.699,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:40.798,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:40.899,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:25:41.002,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:41.107,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:41.214,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:41.316,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:41.417,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:41.517,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:41.617,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:41.722,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:41.827,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:25:41.936,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:42.047,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:42.153,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:42.260,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:25:42.363,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:42.464,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:42.569,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:42.677,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:42.783,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:42.884,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:42.983,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:25:43.084,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:43.194,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:43.302,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:43.407,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:43.514,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:43.620,6d:73:5a:f3:7b:98,0,-42
2019-09-25T10:25:43.730,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:43.833,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:44.048,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:44.149,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:25:44.257,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:44.366,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:44.475,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:44.582,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:44.789,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:44.897,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:25:45.001,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:45.102,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:25:45.206,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:25:45.307,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:25:45.522,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:25:45.623,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:25:45.732,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:25:45.835,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:02.998,6d:73:5a:f3:7b:98,0,-43
2019-09-25T10:26:03.107,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:03.214,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:03.322,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:26:03.428,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:03.531,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:03.638,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:03.740,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:03.840,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:03.946,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:04.050,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:04.159,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:04.258,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:04.370,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:04.478,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:04.587,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:04.688,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:04.789,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:04.889,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:04.999,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:05.105,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:05.311,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:26:05.312,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:05.420,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:05.522,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:05.629,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:05.729,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:05.841,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:05.946,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:06.052,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:06.160,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:06.268,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:06.369,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:06.572,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:06.675,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:06.782,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:06.997,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:07.098,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:07.206,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:26:07.314,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:26:07.423,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:07.525,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:07.626,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:07.728,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:07.828,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:07.931,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:08.037,6d:73:5a:f3:7b:98,0,-46
2019-09-25T10:26:08.148,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:08.248,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:26:08.460,6d:73:5a:f3:7b:98,0,-45
2019-09-25T10:26:08.566,6d:73:5a:f3:7b:98,0,-44
2019-09-25T10:26:08.669,6d:73:5a:f3:7b:98,0,-45
