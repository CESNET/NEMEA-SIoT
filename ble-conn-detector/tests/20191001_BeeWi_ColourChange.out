2019-09-25T10:26:02.998,6d:73:5a:f3:7b:98,0,17163,0,"The device 6d:73:5a:f3:7b:98 was in use for 17163us."
